docs/extendedusage.rst

```diff
@@ -198,6 +200,23 @@ Traits
     'The index of the first track to return'
 
 
+RAML1.0 Types
+~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> api.types
+    {'Person': ObjectType(name='Person', properties={'name': Property(type='string')})}
```

Comment:
what I don't like with OrderedDict is the poor concision of its __repr__
I think a key feature of ramlfication is its nice __repr__ of the parsed raml, which allows people to write their own parsers with print and REPL debugging.
My followup patch will contain a custom OrderedDict with a __repr__ which is very similar as dict, only prefixed with 'o'

 "o{'Person': ObjectType(name='Person', properties=o{'name': Property(type='string')})}"